functions/src/venue.ts

```diff
@@ -424,6 +427,43 @@ export const setAuditoriumSections = functions.https.onCall(
   }
 );
 
+export const addSpaceOwnerBulk = functions.https.onCall(
+  async (data, context) => {
+    assertValidAuth(context);
+
+    const { addedSpacesIds, removedSpacesIds, newOwnerId, worldId } = data;
+
+    await throwErrorIfNotWorldOwner({
+      worldId: worldId,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      userId: context.auth.token.user_id,
```

Comment:
Please don't ignore possible errors and ignore the warnings of ignoring possible errors. Most of the time it's a simple, well... "type massaging", but in this case might be even simpler than that:

```suggestion
      userId: context.auth?.token.user_id,
```